docs/source/references/config.md

```diff
@@ -120,7 +115,7 @@ The following is the default configuration used by Iroha.
 
 `BlockSynchronizer` configuration
 
-Has type `block_sync::Configuration`. Can be configured via environment variable `IROHA_BLOCK_SYNC`
+Has type `Option<block_sync::ConfigurationProxy>`. Can be configured via environment variable `IROHA_BLOCK_SYNC`. More explanation in [this section](#configuration-types).
```

Comment:
```suggestion
Has type `Option<block_sync::ConfigurationProxy>`. Can be configured via environment variable `IROHA_BLOCK_SYNC`. Refer to [configuration types](#configuration-types) for details.
```

same in other places in this file

Reply:
As all this is generated from the above one, choosing this wording